We disable BUILD_NATIVE, as we know little about client's system
and asterisk architecture-dependent optimizaitons are too aggressive

--- build_tools/cflags.xml.1	2017-11-16 20:37:33.569448140 +0000
+++ build_tools/cflags.xml	2017-11-16 20:37:51.779274956 +0000
@@ -130,7 +130,7 @@
		</member>
		<member name="BUILD_NATIVE" displayname="Allow compiler to generate code optimized for the CPU on which the build is performed.">
			<support_level>core</support_level>
-			<defaultenabled>yes</defaultenabled>
+			<defaultenabled>no</defaultenabled>
			<depend>native_arch</depend>
		</member>
	</category>
